services/speech_to_text/v1-corpus-builder.js

```diff
@@ -48,125 +50,192 @@ module.exports = function (RED) {
   }
 
   function executeCreateCustomisation(node, stt, params, msg) {
-    stt.createLanguageModel(params, function (err, response) {
-      node.status({});
-      if (err) {
-        payloadutils.reportError(node, msg, err);
-      } else {
-        msg['customization_id'] = response;
-      }
-      node.send(msg);
+    return new Promise(function resolver(resolve, reject) {
+      stt.createLanguageModel(params)
+        .then((response) => {
+          responseutils.parseResponseFor(msg, response, 'customization_id');
+          resolve();
+        })
+        .catch((err) => {
+          reject(err);
+        })
     });
   }
 
   function executeListCustomisations(node, stt, params, msg) {
-    stt.listLanguageModels(params, function (err, response) {
-      node.status({});
-      if (err) {
-        payloadutils.reportError(node, msg, err);
-      } else {
-        msg['customizations'] = response.customizations ?
-                                      response.customizations: response;
-      }
-      node.send(msg);
+    return new Promise(function resolver(resolve, reject) {
+      stt.listLanguageModels(params)
+        .then((response) => {
+          responseutils.parseResponseFor(msg, response, 'customizations');
+          resolve();
+        })
+        .catch((err) => {
+          reject(err);
+        });
     });
   }
 
   function executeGetCustomisation(node, stt, params, msg) {
-    stt.getLanguageModel(params, function (err, response) {
-      node.status({});
-      if (err) {
-        payloadutils.reportError(node, msg, err);
-      } else {
-        msg['customization'] = response ;
-      }
-      node.send(msg);
+    return new Promise(function resolver(resolve, reject) {
+      stt.getLanguageModel(params)
+        .then((response) => {
+          responseutils.parseResponseFor(msg, response, 'customization');
+          resolve();
+        })
+        .catch((err) => {
+          reject(err);
+        });
     });
   }
 
   function executeDeleteCustomisation(node, stt, params, msg) {
-    stt.deleteLanguageModel(params, function (err, response) {
-      node.status({});
-      if (err) {
-        payloadutils.reportError(node, msg, err);
-      } else {
-        msg['delcustomresponse'] = response ;
-      }
-      node.send(msg);
+    return new Promise(function resolver(resolve, reject) {
+      stt.deleteLanguageModel(params)
+        .then((response) => {
+          msg['delcustomresponse'] = response;
+          resolve();
+        })
+        .catch((err) => {
+          reject(err);
+        })
+    });
+  }
+
+  function executeUpgradeCustomisation(node, stt, params, msg) {
+    return new Promise(function resolver(resolve, reject) {
+      stt.upgradeLanguageModel(params)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 spaces but found 6.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4804371)